wdl/MainVcfQc.wdl

```diff
@@ -0,0 +1,851 @@
+version 1.0
+
+import "CollectQcVcfWide.wdl" as vcfwideqc
+import "CollectQcPerSample.wdl" as persample
+import "CollectSiteLevelBenchmarking.wdl" as sitebench
+import "CollectPerSampleBenchmarking.wdl" as samplebench
+import "TasksMakeCohortVcf.wdl" as MiniTasks
+import "Utils.wdl" as Utils
+
+# Main workflow to perform comprehensive quality control (QC) on
+# an SV VCF output by GATK-SV
+workflow MainVcfQc {
+  input {
+    Array[File] vcfs  # Option to provide a single GATK-SV VCF or an array of position-sharded SV VCFs
+    Array[File] vcf_idxs
```

Comment:
```suggestion
```
I missed this before but requiring index inputs creates some bloating for input files. Can you generate index paths on-the-fly for task calls, e.g. `vcf_index=vcfs[i] + .tbi`?